tests/test_staging/test_staging_helpers.py

```diff
@@ -247,10 +298,91 @@ def test_load_snapshot_feather(self, tmp_path):
         ), "Snapshot df has more columnss than expected."
 
 
-# load_val_snapshot_json [CANT TEST: TOO MANY HARD CODED PATHS]
+class TestLoadValSnapshotJson(object):
+    """Tests for the load_val_snapshot_json function."""
+
+    @patch("src.utils.local_file_mods.load_local_json")
+    @patch("src.staging.validation.validate_data_with_schema")
+    @patch("src.staging.spp_snapshot_processing.full_responses")
+    @patch("src.staging.validation.combine_schemas_validate_full_df")
+    def test_load_val_snapshot_json_correct_response_rate(
+        self,
+        mock_combine_schemas_validate_full_df,
+        mock_full_responses,
```

Comment:
unused

Reply:
As above, I think the mock for the function full_responses is being used